aries-site/src/examples/components/textinput/TextInputValidationExample.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Box, Button, Form, FormField, TextInput } from 'grommet';
+
+export const TextInputValidationExample = () => (
+  <Box>
+    <Box width="small">
+      <Form>
+        <FormField name="username" label="Label" htmlFor="username" required>
+          <TextInput
+            name="username"
+            id="username"
+            placeholder="Placeholder text"
+          />
+        </FormField>
+        <Button label="Submit" type="submit" />
```

Comment:
Thoughts on the word "Validate" as the button label? Thinking that for good UX button text should clearly speak to what action will happen and validate might support this.
```suggestion
        <Button label="Validate" type="submit" />
```